avaframe/com1DFA/com1DFA.py

```diff
@@ -1169,11 +1167,8 @@ def DFAIterate(cfg, particles, fields, dem):
     massEntrained = []
     massTotal = []
 
-    # time stepping scheme info
-    if featLF:
-        log.debug('Use LeapFrog time stepping')
-    else:
-        log.debug('Use standard time stepping')
+    # TODO: add here different time stepping options
```

Comment:
Is this comment actually needed or is it just a reminder for us?

Reply:
more like a reminder for us